src/ray/gcs/tables.cc

```diff
@@ -450,37 +449,37 @@ Status Hash<ID, Data>::Subscribe(const DriverID &driver_id, const ClientID &clie
   return Status::OK();
 }
 
-Status ErrorTable::PushErrorToDriver(const DriverID &driver_id, const std::string &type,
+Status ErrorTable::PushErrorToDriver(const WorkerID &driver_id, const std::string &type,
```

Comment:
This argument can be a 'JobID', which means that we push errors to the driver of the given job.